styles/ClickHouse/CodeblockFences.yml

```diff
@@ -6,7 +6,7 @@ extends: existence
 message: "Instead of '%s' for the code block, use yaml, ruby, plaintext, markdown, javascript, shell, go, python, dockerfile, or typescript."
 link: https://docs.gitlab.com/development/documentation/styleguide/#code-blocks
 vocab: false
-level: warning
```

Comment:
This one I think we should keep as a warning personally, just because there is no syntax highlighting without it